src/pymmcore_widgets/useq_widgets/_positions.py

```diff
@@ -185,16 +198,63 @@ def value(
         ):
             if not r.get(self.NAME.key, True):
                 r.pop(self.NAME.key, None)
-            out.append(useq.Position(**r))
+
+            if not self.use_af.isChecked():
+                pos = useq.Position(**r)
+            else:
+                pos = useq.Position(**r)
```

Comment:
r is a dict of kwargs used to create the position.  rather than creating the position with **r and then fixing it, can you just update the kwargs before creation?